modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js

```diff
@@ -241,8 +241,8 @@ export default class ScenegraphLayer extends Layer {
   draw({moduleParameters = null, parameters = {}, context}) {
     if (!this.state.scenegraph) return;
 
-    if (this.props._animations && this.state.animator) {
-      this.state.animator.animate(context.animationProps.time);
+    if (this.props._animations && this.state.animator && context.deck) {
```

Comment:
This is confusing. I can see `context.deck.animationLoop` being undefined, but I would expect `context.deck` to always be defined?

Reply:
I was confused by this too. The ScenegraphLayer generated layer tests fail indicating `context.deck` is undefined. Is there supposed to be a phase where it's undefined or is it just a test anomoly?

Reply:
There is no deck instance in the layer testing context. It is driven by a standalone `LayerManager` instance.